layers/+distribution/spacemacs/packages.el

```diff
@@ -369,6 +369,7 @@
          :exit t)
         ("n" spacemacs/quick-ahs-forward)
         ("N" spacemacs/quick-ahs-backward)
+        ("p" spacemacs/quick-ahs-backward)
         ("R" ahs-back-to-start)
```

Comment:
@robbyoconnor yes it should be conditional but for micro-states this wasn't possible (and probably isn't yet) But with current transition to hydra and justbur's work on transient-states it should be possible in future.